locales/es/translation.json

```diff
@@ -1,1327 +1,1328 @@
+    "Can manage all aspects of the inventory": "Puede gestionar todos los aspectos del inventario",
+    "Can manage all aspects of the job template": "Puede gestionar todos los aspectos de la plantilla",
+    "Can manage all aspects of the organization": "Puede gestionar todos los aspectos de la organización",
+    "Can manage all aspects of the project": "Puede gestionar todos los aspectos del proyecto",
+    "Can manage all aspects of the workflow job template": "Puede gestionar todos los aspectos del modelo de trabajo del flujo de trabajo",
+    "Can manage all credentials of the organization": "Puede gestionar todas las credenciales de la organización",
+    "Can manage all execution environments of the organization": "Puede gestionar todos los entornos de ejecución de la organización",
+    "Can manage all inventories of the organization": "Puede gestionar todos los inventarios de la organización",
+    "Can manage all job templates of the organization": "Puede gestionar todos los modelos de trabajo de la organización",
+    "Can manage all notifications of the organization": "Puede gestionar todas las notificaciones de la organización",
+    "Can manage all projects of the organization": "Puede gestionar todos los proyectos de la organización",
+    "Can manage all workflows of the organization": "Puede gestionar todos los flujos de trabajo de la organización",
+    "can see all aspects of the systems automation, but has no permission to run or change automation": "puede ver todos los aspectos de la automatización de los sistemas, pero no tiene permiso para ejecutar o cambiar la automatización",
+    "Can use the credential in a job template": "Puede utilizar la credencial en una plantilla de trabajo",
+    "Can use the inventory in a job template": "Puede utilizar el inventario en una plantilla de trabajo",
+    "Can use the project in a job template": "Puede utilizar el proyecto en una plantilla de trabajo",
+    "Can view all aspects of the organization": "Puede ver todos los aspectos de la organización",
+    "Cancel": "Cancelar",
+    "Cancel job": "Cancelar tarea",
+    "Cancel jobs_one": "Cancelar tarea",
+    "Cancel jobs_other": "Cancelar tareas",
+    "Cancel project sync": "Cancelar la sincronización del proyecto",
+    "Cancel project sync_one": "Cancelar la sincronización del proyecto",
+    "Cancel project sync_other": "Cancelar la sincronización del proyecto",
+    "Cancel selected jobs": "Cancelar tareas seleccionadas",
+    "Cancel selected projects": "Cancelar proyectos seleccionados",
+    "Canceled": "Cancelado",
+    "Cannot sync project": "No se puede sincronizar el proyecto",
+    "Capacity": "Capacidad",
+    "Certified": "Certificado",
+    "Check": "Comprobar",
+    "check out our instruct guides": "consulte nuestras guías de instrucciones",
+    "Choices": "Opciones",
+    "Click to view latest project sync job": "Haga clic para ver la última sincronización de tarea del proyecto",
```

Comment:
I think this should be "Haga clic para ver la última tarea de sincronización del proyecto"